pulp_container/tests/functional/api/test_rbac_remotes.py

```diff
@@ -0,0 +1,139 @@
+# coding=utf-8
+"""Tests that container remotes have RBAC."""
+from random import choice
+import unittest
+
+from pulp_smash import utils
+from pulp_smash.pulp3.bindings import monitor_task
+from pulp_smash.pulp3.constants import ON_DEMAND_DOWNLOAD_POLICIES
+
+from pulp_container.tests.functional.utils import (
+    del_user,
+    gen_container_remote,
+    gen_user,
+    skip_if,
+)
+
+from pulpcore.client.pulp_container.exceptions import ApiException
+
+
+class RBACRemotesTestCase(unittest.TestCase):
+    """RBAC remotes."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create class-wide variables and prepare api users."""
+        cls.user1 = gen_user(["container.add_containerremote"])
+        cls.user2 = gen_user(["container.view_containerremote"])
```

Comment:
does it makes sense to add one more test in this PR that checks user2 cannot create remotes?